system/netdata.logrotate.in

```diff
@@ -0,0 +1,13 @@
+@localstatedir_POST@/log/netdata/*.log {
+	daily
+	missingok
+	rotate 14
+	compress
+	delaycompress
+	notifempty
+	sharedscripts
+	postrotate
+		systemctl restart netdata.service
```

Comment:
Yeah, looking at that as part of adding an init script, as it's no good for wheezy.
copytruncate is hideous, /etc/init.d/netdata restart should work on most things though (even Debian jessie, where it will be intercepted and translated to a systemd call)